clippy.toml

```
@@ -1,2 +1,2 @@
-doc-valid-idents = ["RustSAT", "CaDiCaL", "MaxSAT", ".."]
+doc-valid-idents = ["RustSAT", "CaDiCaL", "MaxSAT", "..", "BatSat"]
```

```suggestion
doc-valid-idents = ["RustSAT", "CaDiCaL", "MaxSAT", "BatSat", ".."]
```
Not entirely sure how clippy handles these, but since the `".."` means "extend with clippies default list", let's better leave this at the end.